CHANGES.md

```diff
@@ -10,6 +10,7 @@
 - Add partial support for the match statement. As it's experimental, it's only enabled
   when `--target-version py310` is explicitly specified (#2586)
 - Add support for parenthesized with (#2586)
+- Fixed Vim with Python 3.10 by removing deprecated distutils (#2610)
```

Comment:
```suggestion

### Integrations

- Fixed vim plugin with Python 3.10 by removing deprecated distutils import (#2610)
```

See 21.10b0 for previous vim changes, we should try to keep the wording consistent.